notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<h3>Retrieve prediction results </h3><p> </p><p>After the batch job completes, the results are written to the Cloud Storage output bucket you specified in the batch request. You call the method <code>iter_outputs()</code> to get a list of each Cloud Storage file generated with the results.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>